index.md

```diff
@@ -32,76 +53,43 @@ Event
-:link: https://github.com/pymc-devs/pymc/wiki/GSoC-2022-projects
-:class-header: bg-pymc-two
-
-Internships
-^^^
-If you are interested in open source, GSoC is the perfect opportunity to dive in and get involved.
-This year, anyone new to open source and over 18 years can participate!
-:::
 :::{grid-item-card} Sprint testimonials
 :link: sprint_testimonial
 :link-type: ref
-:class-header: bg-pymc-three
+:class-header: bg-pymc-one
 
 Blog post
 ^^^
 Read about the recent PyMC-Data Umbrella sprint in this interview with
 Sandra Meneses, one of the participants who submitted a PR
 :::
-:::{grid-item-card} PyMC 4.0 is closing in
-:link: v4_announcement
-:link-type: ref
-:class-header: bg-pymc-one
 
-Release announcement
-^^^
-Check out this blog post for a sneak preview of all the improvements PyMC 4.0 will bring!
-Or install the already available beta release and try it for yourself!
-:::
 ::::
 :::::
 
-## The PyMC library
+# The PyMC project
+PyMC is a community driven project with the goal of making Bayesian modeling
+and probabilistic programming intuitive and performant.
 
-{doc}`PyMC <pymc:index>` is a probabilistic programming library for Python that allows users
-to fit Bayesian models using a variety of numerical methods,
-most notably Markov chain Monte Carlo (MCMC) and variational inference (VI).
-Its flexibility and extensibility make it applicable to a large suite of problems.
-Along with core model specification and fitting functionality,
-PyMC integrates with {doc}`ArviZ <arviz:index>` for exploratory analysis of the results.
+The flagship of the PyMC project is the PyMC library, but PyMC also coordinates
+many other activities:
 
-### Features
-PyMC strives to make Bayesian modeling as simple and painless as possible,
-allowing users to focus on their scientific problem, rather than on the methods used to solve it.
-Here is a partial list of its features:
-
-* Modern methods for fitting Bayesian models, including MCMC and VI.
-* Includes a large suite of well-documented statistical distributions.
-* Uses {doc}`Aesara <aesara:index>` as the computational backend, allowing for fast expression evaluation, automatic gradient calculation, and GPU computing.
-* Built-in support for Gaussian process modeling.
-* Extensible: easily incorporates custom step methods and unusual probability distributions.
-* Bayesian models can be embedded in larger programs, and results can be analyzed with the full power of Python.
+* Organizing [PyMCon](https://pymcon.com)
+* Curating the {doc}`PyMC example gallery <nb:index>`
+* Answer questions and moderate discussions on [PyMC Discourse](https://discourse.pymc.io/)
+* Translating the code examples of Bayesian statistics books in
+  [PyMC resources](https://github.com/pymc-devs/pymc-resources)
+* [Office hours](https://discourse.pymc.io/tag/office-hours) and
+  [sprints](https://pymc-data-umbrella.xyz/en/latest/) to encourage people to contribute to open source
```

Comment:
```suggestion
  [Sprints](https://pymc-data-umbrella.xyz/en/latest/) to encourage people to contribute to open source
```

Reply:
I think this was correct, middle of the sentence.

Reply:
oh, its correct I thought it was a new bullet point, but now I want to ask why not make it a new bullet point? Either is fine just curious